src/scripts/main.js

```diff
@@ -1,5 +1,31 @@
 'use strict';
 
+const spiderElement = document.querySelector('.spider');
+
 document.addEventListener('click', (e) => {
   // write code here
+  e.preventDefault();
+
+  const wall = e.target.closest('.wall');
+  const wallCords = wall.getBoundingClientRect();
```

Comment:
The code attempts to access `wall.getBoundingClientRect()` before checking if `wall` is null. This can lead to a runtime error if the click event does not occur on an element with the class 'wall'. Consider moving the null check for `wall` before accessing its properties.